### ISSUE TITLE
Use consistent wording for "out of memory" / "memory allocation failed" error messages

### ISSUE THREAD

**Issue by TimWolla** [haproxy/haproxy#1025]
Currently we have a mix of:

- `Out of memory error.`
- `Out of memory error`
- `out of memory error`
- `out of memory`
- `out of memory.`
- `out of memory [some context]`

All these should be adjusted to use the same wording for consistency and it should also make the executable a little bit smaller :smile:

**Comment by wtarreau** [comment]
I think it could slightly depend on the context. For example at the beginning of a sentence you'll have an upper case but not in the middle. Similarly there might be some occurrences of "error: %s" where it's better to have "error: out of memory" than "error: out of memory error". But I sincerely doubt we have that many different contexts and it's likely that 2 or 3 at most would be needed.

**Comment by TimWolla** [comment]
Yes, it's not a 100% mindless search and replace. I did look into a few places while writing up this issue: Most of them have exactly the same context (usually config parsing) and are worded *slightly* differently. Adjusting all the ones in the config parser and similar is a no brainer.

**Comment by breakerbar** [comment]
Working on this, should have a patch in a day or so -- there's a lot of instances, as you mentioned.

**Comment by TimWolla** [comment]
@breakerbar No PR, please. We're not using PRs. See: https://github.com/haproxy/haproxy/blob/6c940569f607a2311861358fc607a4a08ce028c3/CONTRIBUTING#L933-L936

**Comment by theNKCode** [comment]
Hey as nobody get assigned this task, I am glad to do this. As much I understand You want consistent wording for "out of memory Out of memory error, Out of memory error, out of memory error, out of memory, out of memory, out of memory [some context], memory allocation failed" error messages.   Please assign me this task. and comment what is most expected wording for you.

**Comment by wtarreau** [comment]
@theNKCode do not hesitate to propose something. As Tim and I mentioned above, let's just be careful not to drop any context when there is one. But at least the 5 first forms could likely just become "Out of memory".

**Comment by iamkishan98** [comment]
Hi @TimWolla, I am new to open source contributions. I would love to fix this issue. Can you please assign it to me ?

**Comment by wtarreau** [comment]
Hi! Thanks for willing to participate. We don't "assign" issues. The simple fact that you expressed your interest is sufficient to let others know you're working on it. However please have a long read of the CONTRIBUTING file of the project before engaging into this, it will save you lots of time!
